swarm/network/simulation/kademlia_test.go

```diff
@@ -28,6 +28,8 @@ import (
 )
 
 func TestWaitTillHealthy(t *testing.T) {
+
+	t.Skip("temporarily disabled as simuations.WaitTillHealthy cannot be trusted")
```

Comment:
minor nitpicking: typo in text (simuations)

Reply:
copy/paste with typo <3